src/lib/swapper/swappers/CowSwapper/getCowSwapMinMax/getCowSwapMinMax.test.ts

```diff
@@ -1,23 +1,34 @@
-import { BTC, ETH, FOX, WETH } from '../../utils/test-data/assets'
+import { KnownChainIds } from '@shapeshiftoss/types'
+
+import { BTC, ETH, FOX, USDC_GNOSIS, WETH, XDAI } from '../../utils/test-data/assets'
 import { MAX_COWSWAP_TRADE } from '../utils/constants'
 import { getCowSwapMinMax } from './getCowSwapMinMax'
 
 jest.mock('state/zustand/swapperStore/amountSelectors', () => ({
   ...jest.requireActual('state/zustand/swapperStore/amountSelectors'),
   selectSellAssetUsdRate: jest.fn(() => '0.25'),
 }))
+const supportedChainIds = [KnownChainIds.EthereumMainnet, KnownChainIds.GnosisMainnet]
 
 describe('getCowSwapMinMax', () => {
   it('returns minimumAmountCryptoHuman and maximumAmountCryptoHuman', () => {
-    const maybeMinMax = getCowSwapMinMax(FOX, WETH)
+    const maybeMinMax = getCowSwapMinMax(FOX, WETH, supportedChainIds)
     expect(maybeMinMax.isErr()).toBe(false)
     const minMax = maybeMinMax.unwrap()
     expect(minMax.minimumAmountCryptoHuman).toBe('80')
     expect(minMax.maximumAmountCryptoHuman).toBe(MAX_COWSWAP_TRADE)
   })
 
   it('returns minimumAmountCryptoHuman and maximumAmountCryptoHuman for ETH as buy asset', () => {
-    const maybeMinMax = getCowSwapMinMax(FOX, WETH)
+    const maybeMinMax = getCowSwapMinMax(FOX, WETH, supportedChainIds)
+    expect(maybeMinMax.isErr()).toBe(false)
+    const minMax = maybeMinMax.unwrap()
+    expect(minMax.minimumAmountCryptoHuman).toBe('80')
+    expect(minMax.maximumAmountCryptoHuman).toBe(MAX_COWSWAP_TRADE)
+  })
+
+  it('returns minimumAmountCryptoHuman and maximumAmountCryptoHuman for USDC on Gnosis as buy asset', () => {
```

Comment:
This is testing Gnosis USDC as a sell asset, not as a buy asset
```suggestion
  it('returns minimumAmountCryptoHuman and maximumAmountCryptoHuman for XDAI as buy asset', () => {
```

Alternatively, since the expected output is the exact same for both, can consolidate both tests in the previously existing one and use the `native asset` terminology